src/reducers/rootReducer.js

```diff
@@ -0,0 +1,24 @@
+import { combineReducers } from 'redux';
+import { gameTagsReducer } from './gameTagsReducer';
+import { imageIndexReducer } from './imageIndexReducer';
+import { submitTagsReducer } from './submitTagsReducer';
+import { currentImageReducer } from './currentImageReducer';
+import { isTagModeReducer } from './isTagModeReducer';
+import { gameScoreReducer } from './gameScoreReducer';
+import { networkErrorReducer } from './networkErrorReducer';
+import { currentGridReducer } from './currentGridReducer';
+import { tagsIndexReducer } from './tagsIndexReducer';
+import { isViewingTagsReducer } from './isViewingTagsReducer';
+
+export const rootReducer = combineReducers({
+  error: networkErrorReducer,
+  index: imageIndexReducer,
+  tags: gameTagsReducer,
+  submitError: submitTagsReducer,
+  currentImage: currentImageReducer,
+  isTagMode: isTagModeReducer,
+  gameScore: gameScoreReducer,
+  currentGrid: currentGridReducer,
+  isViewingTags: isViewingTagsReducer,
+  searchedTag: tagsIndexReducer,
```

Comment:
hypothetical question -- what would you do if there are multiple types of data that need to be fetched, and also multiple types of data that can be `POST`ed?

Reply:
I probably would group each type of data in its own reducer, instead of breaking it all down one by one.